yarn.lock

```diff
@@ -28,6 +33,28 @@
   version "5.5.0"
   resolved "https://registry.yarnpkg.com/@types/semver/-/semver-5.5.0.tgz#146c2a29ee7d3bae4bf2fcb274636e264c813c45"
 
+"@types/unist@*", "@types/unist@^2.0.0":
+  version "2.0.3"
+  resolved "https://registry.yarnpkg.com/@types/unist/-/unist-2.0.3.tgz#9c088679876f374eb5983f150d4787aa6fb32d7e"
+  integrity sha512-FvUupuM3rlRsRtCN+fDudtmytGO6iHJuuRKS1Ss0pG5z8oX0diNEw94UEL7hgDbpN94rgaK5R7sWm6RrSkZuAQ==
```

Comment:
Caused by new dependency `@types/unist`, tests are failed... 😭 

https://travis-ci.org/TypeStrong/ts-loader/jobs/498777265#L544

Reply:
The reason for errors is that `remark` depends on `@types/unist`, which is using `unknown` type since TypeScript 3.

```
$ npm ls @types/unist
ts-loader@5.3.3 /Users/koba/git/ybiquitous/ts-loader
└─┬ remark-cli@6.0.1
  └─┬ remark@10.0.1
    └─┬ unified@7.1.0
      ├── @types/unist@2.0.3
      └─┬ @types/vfile@3.0.2
        ├── @types/unist@2.0.3  deduped
        └─┬ @types/vfile-message@1.0.1
          └── @types/unist@2.0.3  deduped
```

Does anyone have a good idea? Unless, I should give up using `remark`...

Reply:
Could `@types/unist` be amended not to use `unknown`? I'm a Definitely Typed member - I can review a PR if you raise it?

Reply:
Thanks for the response!

Sadly I don't know how to fix `unknown`. 🤷‍♂️ 
But I found the issue in unifiedjs/unified#45 which may resolve this problem, so I posted a [comment](https://github.com/unifiedjs/unified/issues/45#issuecomment-485234640) to the issue.

I hope strongly the issue will be fixed, but if it will not be achieved, then it's OK to close the PR. 😓

Reply:
Existing types could be copied into a `ts3.1` directory, while types in the root can have `unknown` replaced by `any`, then update the package.json like this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2194a1ac8b3e83a882b07257a8dfdc94b5fe2b72/types/jsdom/package.json#L7

Reply:
@andrewbranch Thanks for the advice! I found the same issue ⬇️ 

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/31470

But sadly, it seems that the discussion is not advanced much. 😞

Reply:
The latest version of `unified` (v8.1.0) has solved the problem! I'll rebase this PR and try again soon! 💪 

```
$ npm v unified@latest dependencies

{
  bail: '^1.0.0',
  extend: '^3.0.0',
  'is-plain-obj': '^2.0.0',
  trough: '^1.0.0',
  vfile: '^4.0.0'
}
```

Reply:
There is a problem yet sadly... 😞 

I reported the following issue for the `vfile` which may cause a compile error. Let's see how it goes.
https://github.com/vfile/vfile/issues/36